python/GafferSceneUITest/ParameterInspectorTest.py

```diff
@@ -558,12 +565,47 @@ def testNonExistentLocation( self ) :
 	def testNonExistentAttribute( self ) :
 
 		light = GafferSceneTest.TestLight()
+
+		editScope = Gaffer.EditScope()
+		editScope.setup( light["out"] )
+		editScope["in"].setInput( light["out"] )
+
 		self.assertIsNone( self.__inspect( light["out"], "/light", "exposure", attribute = "nothingHere" ) )
+		self.assertIsNone( self.__inspect( light["out"], "/light", "exposure", editScope, attribute = "nothingHere" ) )
 
 	def testNonExistentParameter( self ) :
 
 		light = GafferSceneTest.TestLight()
+
+		editScope = Gaffer.EditScope()
+		editScope.setup( light["out"] )
+		editScope["in"].setInput( light["out"] )
+
 		self.assertIsNone( self.__inspect( light["out"], "/light", "nothingHere" ) )
+		self.assertIsNone( self.__inspect( light["out"], "/light", "nothingHere", editScope ) )
```

Comment:
Should this be `__inspect( editScope["out"], ... )` as well?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5416/commits/47f0781834fb8e5d80c2a458574ba1a92d3d9a28